src/cli/cli_config.rs

```diff
@@ -112,6 +113,18 @@ pub struct LockFileUpdateConfig {
     pub lock_file_usage: super::LockFileUsageConfig,
 }
 
+#[derive(Parser, Debug, Default, Clone)]
+#[clap(next_help_heading = consts::CLAP_SOLVER_OPTIONS)]
+pub struct SolverConfig {
+    /// Exclude any packages that have been created after the given date.
+    #[clap(
+        long = "exclude-newer",
+        env = "PIXI_EXCLUDE_NEWER",
+        value_name = "DATE"
+    )]
+    pub exclude_newer: Option<DateTime<Utc>>,
```

Comment:
Should this be conda and in the future pypi specific?

Reply:
I want to add pypi in the next PR but it will use the same CLI.